crates/stdarch-gen/neon.spec

```diff
@@ -1544,6 +1544,28 @@ validate 9, 30, 11, 20, 13, 18, 15, 48
 aarch64 = pmull
 generate poly8x16_t:poly8x16_t:poly16x8_t
 
+/// Floating-point fused Multiply-Add to accumulator(vector)
+name = vfma
+no-q
+a = 2.0, 3.0
+b = 6.0, 4.0
+c = 8.0, 18.0
+validate 20.0, 30.0
+
+target = fp-armv8
+arm = vfma
+aarch64 = fma
+link-arm = llvm.fma._EXT_
+link-aarch64 = llvm.fma._EXT_
+generate float64x1_t
```

Comment:
The `f64` versions of this intrinsic are only available on aarch64 according to clang and gcc. The `float64x1_t` type is also aarch64-only.

Reply:
Done. Thanks.

Reply:
> The `f64` versions of this intrinsic are only available on aarch64 according to clang and gcc. The `float64x1_t` type is also aarch64-only.

Done. Thanks.